### PR TITLE
Paused prompt fixes and tests

### PR DESCRIPTION
**Pull request by gabriel**
This PR mainly adds testing for the paused prompt.
Includes scripts which mock prompt and keybase check-in-use behavior.

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/updater-hackers

**Comment by gabriel** [comment]
bump

File: keybase/prompt_test.go [hunk 1]
```diff
@@ -96,3 +96,31 @@ func TestPromptError(t *testing.T) {
...
+
+func testPausedPromptWithCommand(t *testing.T, promptCommand string, timeout time.Duration) (bool, error) {
+	cfg, _ := testConfig(t)
+	ctx := newContext(&cfg, log)
+	assert.NotNil(t, ctx)
+
+	if timeout > 0 {
+		defaultPromptTimeout := promptTimeout
+		promptTimeout = timeout
+		defer func() { promptTimeout = defaultPromptTimeout }()
```
Comment (jzila) [inline keybase/prompt_test.go]: this kind of ruins the ability to run these tests in parallel

**Comment by jzila** [comment]
Mostly LGTM but it looks like there's a conflict.

File: keybase/prompt_test.go (hunk 1 above)
Comment (gabriel) [inline keybase/prompt_test.go]: Yeah this is smelly... removing the prompt global timeout vars, and making an arg like a sane person :)

**Pull request merged**